### DISCUSSION TITLE
输出插件对Opensearch的支持

### DISCUSSION THREAD

**Discussion by Sic9527** [alibaba/ilogtail#1628] [Help]
由于ElasticSearch的开源协议原因，使用Opensearch代替，logtail的ElasticSearch输出插件对Opensearch存在不支持情况
[ERR] [flusher_elasticsearch.go:223] [Flush] [file_simple/2]	AlarmType:FLUSHER_FLUSH_ALARM	flush elasticsearch request fail, error:the client noticed that the server is not Elasticsearch and we do not support this unknown product

**Comment by messixukejia** [comment]
欢迎一起来参与开发😊
